src/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/DirectoryRequest.cs

```diff
@@ -564,79 +308,42 @@ public int SizeLimit
 
         public TimeSpan TimeLimit
         {
-            get
-            {
-                return _directoryTimeLimit;
-            }
-
+            get => _directoryTimeLimit;
             set
             {
                 if (value < TimeSpan.Zero)
                 {
-                    throw new ArgumentException(String.Format(CultureInfo.CurrentCulture, SR.NoNegativeTime), "value");
+                    throw new ArgumentException(SR.NoNegativeTime, nameof(value));
                 }
 
-                // prevent integer overflow
-                if (value.TotalSeconds > Int32.MaxValue)
-                    throw new ArgumentException(String.Format(CultureInfo.CurrentCulture, SR.TimespanExceedMax), "value");
+                // Prevent integer overflow.
+                if (value.TotalSeconds > int.MaxValue)
+                {
+                    throw new ArgumentException(SR.TimespanExceedMax, nameof(value));
+                }
 
                 _directoryTimeLimit = value;
             }
         }
 
-        public bool TypesOnly
-        {
-            get
-            {
-                return _directoryTypesOnly;
-            }
-
-            set
-            {
-                _directoryTypesOnly = value;
-            }
-        }
+        public bool TypesOnly { get; set; }
 
-        //
-        // Private/protected
-        //
-
-        private string _dn = null;
-        private StringCollection _directoryAttributes = new StringCollection();
         private object _directoryFilter = null;
         private SearchScope _directoryScope = SearchScope.Subtree;
         private DereferenceAlias _directoryRefAlias = DereferenceAlias.Never;
         private int _directorySizeLimit = 0;
         private TimeSpan _directoryTimeLimit = new TimeSpan(0);
-        private bool _directoryTypesOnly = false;
     }
 }
 
 namespace System.DirectoryServices.Protocols
 {
-    using System;
-
     public class DsmlAuthRequest : DirectoryRequest
     {
-        private string _directoryPrincipal = "";
-
         public DsmlAuthRequest() { }
 
-        public DsmlAuthRequest(string principal)
-        {
-            _directoryPrincipal = principal;
-        }
+        public DsmlAuthRequest(string principal) => Principal = principal;
 
-        public string Principal
-        {
-            get
-            {
-                return _directoryPrincipal;
-            }
-            set
-            {
-                _directoryPrincipal = value;
-            }
-        }
+        public string Principal { get; set; } = string.Empty;
```

Comment:
Wouldn't it be better to do the string.IsEmpty assignment inside of the `public DsmlAuthRequest()` ctor?